charts/hivemq-swarm/values.yaml

```diff
@@ -0,0 +1,240 @@
+  ##
+  affinity: {}
+    # podAntiAffinity:
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     # Try not to co-locate with other swarm instances
+    #     - weight: 50
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: swarm-task
+    #               operator: In
+    #               values:
+    #                 - ${LABEL}
+    #         topologyKey: "kubernetes.io/hostname"
+    #     # Try not to co-locate with HiveMQ nodes
+    #     - weight: 60
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: app
+    #               operator: In
+    #               values:
+    #                 - hivemq
+    #         topologyKey: "kubernetes.io/hostname"
+  # Add environment variables for all agents
+  env: [ ]
+
+  ## Additional volumes for the HiveMQ Swarm agent pod.
+  extraVolumes: [ ]
+
+  ## Extra volume mounts that will be added to the swarm agent container
+  extraVolumeMounts: [ ]
+
+## Monitoring configuration
+##
+monitoring:
+  # Deploy ServiceMonitor for Prometheus Operator and enable Prometheus
+  enabled: false
+  # Deploy a dedicated instance of the prometheus operator, including grafana, as sub-chart
+  dedicated: false
+  # dashboardNamespace: monitoring (defaults to use the namespace this chart is deployed to)
+  serviceMonitor:
+    interval: 15s
+    #  namespace: monitoring  (defaults to use the namespace this chart is deployed to)
+    scrapeTimeout: 10s
+  
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+## Pod annotations
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+podAnnotations: {}
+
+## Configure the service to access toe HiveMQ Swarm commander
+## ref: https://kubernetes.io/docs/concepts/services-networking/service/
+##
+service:
+  type: ClusterIP
+  port: 80
+  nodePort: ""
+
+## Configure the ingress resource that allows you to access the HiveMQ Swarm commander
+## ref: http://kubernetes.io/docs/user-guide/ingress/
+##
+ingress:
+  ## Set to true to enable ingress record generation
+  ##
+  enabled: false
+
+  ## Ingress annotations done as key:value pairs
+  ##
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+
+  ## The list of hostnames to be covered with this ingress record.
+  ##
+  hosts:
+    - host: chart-example.local
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+## Node labels for pod assignment. Evaluated as a template.
+## ref: https://kubernetes.io/docs/user-guide/node-selection/
+##
+nodeSelector: {}
```

Comment:
Just noticed this - we set this here "globally" for both agent and commander - maybe we should move this into the individual sections?